package/yast2-bootloader.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Jun  1 14:20:30 UTC 2015 - jreidinger@suse.com
+
+- do not crash if last tab do not exists like for non-supported
+  bootloader (bnc#923458)
```

Comment:
it should for unsupported bootloader that it do not show red popup with crash :)